components/Header/index.tsx

```diff
@@ -19,7 +19,9 @@ interface Props {
 
 const Header: FunctionComponent<Props> = props => (
     <>
-        <Column className='flex-large'>
+        <Column className={props.isHomepage 
```

Comment:
We need to toggle the `width` property in our header between the homepage and tag templates.